Makefile

```diff
@@ -109,8 +139,10 @@ GO_BUILD_DEBUG := $(GO_XCOMPILE) $(GO) build --buildmode=exe -trimpath $(EXTRA_F
 		-tags "$(BUILDTAGS)" -gcflags="all=-N -l" -ldflags "${LDFLAGS}"
 
 RUN_TEST_CONT := $(CONTAINER_ENGINE) run ${DOCKER_RUN_PROXY} \
-		-t --privileged --rm \
-		-v $(CURDIR):$(RUNC)                                 \
+		-t --privileged --rm                         \
```

Comment:
Based on the assumptions I made when extending Sysbox support to ARM64, test execution is not allowed in cross-compilation scenarios, so I don't think we need to carry these ARCH variables inside the UT container (docker will pull the proper image based on the current system architecture).

Btw, this was working after ARM4 support changes were merged.

Reply:
You are correct; let me double check why I added them. If they aren't needed I'll remove them.

Reply:
It's actually needed, because the TARGET_ARCH env var is used by sysbox-runc/tests/integration/helper.bash:

```
RUNC_BUILDDIR=${RUNC_BUILDROOT}/${TARGET_ARCH} 
```

Thus, this variable needs to be passed into the sysbox-runc unit-test container.

Reply:
Got it.